setup.sh

```diff
@@ -0,0 +1,281 @@
+#!/bin/bash
+# Bash Menu Script Example
+InstallFilesFolder=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+MailScannerVersion="5.0.3-7"
+
+if cat /etc/*release | grep ^NAME | grep CentOS; then
+    OS="CentOS"
+    PM="yum"
+    MailScannerDownloadPath="https://s3.amazonaws.com/msv5/release/MailScanner-$MailScannerVersion.rhel.tar.gz"
+elif cat /etc/*release | grep ^NAME | grep Red; then
+    OS="RedHat"
+    PM="yum"
+    MailScannerDownloadPath="https://s3.amazonaws.com/msv5/release/MailScanner-$MailScannerVersion.rhel.tar.gz"
+elif cat /etc/*release | grep ^NAME | grep Fedora; then
+    OS="Fedora"
+    PM="yum"
+    MailScannerDownloadPath="https://s3.amazonaws.com/msv5/release/MailScanner-$MailScannerVersion.rhel.tar.gz"
+elif cat /etc/*release | grep ^NAME | grep Ubuntu; then
+    OS="Ubuntu"
+    PM="apt-get"
+    MailScannerDownloadPath="https://s3.amazonaws.com/msv5/release/MailScanner-$MailScannerVersion.deb.tar.gz"
+elif cat /etc/*release | grep ^NAME | grep Debian ; then
+    OS="Debian"
+    PM="apt-get"
+    MailScannerDownloadPath="https://s3.amazonaws.com/msv5/release/MailScanner-$MailScannerVersion.deb.tar.gz"
+else
+    echo "OS NOT SUPPORTED - Please perform a manual install"
+    exit 1;
+fi
+
+EndNotice=""
+Webuser="www-data"
```

Comment:
This should be set when we check for installed web servers or select the web server we want.

Also need to make sure this is passed to the other scripts that currently use "www-data"